lab_1_keywords_tfidf/main.py

```diff
@@ -93,16 +62,13 @@ def remove_stop_words(tokens: list[str], stop_words: list[str]) -> Optional[list
 
     In case of corrupt input arguments, None is returned
     """
-    my_tokens = []
-    if not (check_list(tokens, str, False) and check_list(stop_words, str, True)):
+    if not (isinstance(tokens, list) and isinstance(stop_words, list)):
         return None
-    for token in tokens:
-        if token not in stop_words:
-            my_tokens.append(token)
-    return my_tokens
+
+    return [word for word in tokens if word not in stop_words]
 
 
-def calculate_frequencies(tokens: list[str]) -> Optional[dict[str, int]]:
+def calculate_frequencies(text_without_stopwords: list[str]) -> Optional[dict[str, int]]:
```

Comment:
Why did you change the interface?

Reply:
That's because of refractor :(
Name 'tokens' seemed not really meaningful to me, I was not thinking of it as of changing the interface

Reply:
changed